### PR TITLE
[zh] Sync control-plane-node-communication

### PR DESCRIPTION
**Pull request by tengqm**
The original master-node-communication page was renamed with some new texts added about Konnectivity. This PR resync the Chinese localization with English text added back as comments.

### CODE REVIEW

**Comment by tengqm** [comment]
preview link: https://deploy-preview-21913--kubernetes-io-master-staging.netlify.app/zh/docs/concepts/architecture/control-plane-node-communication/

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+One or more forms of [authorization](/docs/reference/access-authn-authz/authorization/) should be enabled, especially if [anonymous requests](/docs/reference/access-authn-authz/authentication/#anonymous-requests) or [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens) are allowed.
+-->
+
+## 节点到控制面
+
+Kubernetes 采用的是中心辐射型（Hub-and-Spoke）API 模式。
+所有从集群（或所运行的 Pods）发出的 API 调用都终止于 apiserver（其它控制面组件都没有被设计为可暴露远程服务）。
+apiserver 被配置为在一个安全的 HTTPS 端口（443）上监听远程连接请求，
+并启用一种或多种形式的客户端[身份认证](/docs/reference/access-authn-authz/authentication/)机制。
+一种或多种客户端[鉴权机制](/docs/reference/access-authn-authz/authorization/)机制应该被启用，
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:41]:
```suggestion
一种或多种客户端[鉴权机制](/docs/reference/access-authn-authz/authorization/)应该被启用，
```

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+特别是在允许使用[匿名请求](/docs/reference/access-authn-autha/authentication/#anonymous-requests)
+或[服务账号令牌](/docs/reference/access-authn-authz/authentication/#service-account-tokens)的时候。
+
+<!--
+Nodes should be provisioned with the public root certificate for the cluster such that they can connect securely to the apiserver along with valid client credentials. For example, on a default GKE deployment, the client credentials provided to the kubelet are in the form of a client certificate. See [kubelet TLS bootstrapping](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/) for automated provisioning of kubelet client certificates.
+-->
+
+应该使用集群的公共根证书开通节点，这样它们就能够基于有效的客户端凭据安全地连接 apiserver。
+例如：在一个默认的 GCE 部署中，客户端凭据以客户端证书的形式提供给 kubelet。
+请查看[kubelet TLS 启动引导](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/)
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:51]:
```suggestion
请查看 [kubelet TLS 启动引导](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/)
```

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+以了解如何自动提供 kubelet 客户端证书。
+
+<!--
+Pods that wish to connect to the apiserver can do so securely by leveraging a service account so that Kubernetes will automatically inject the public root certificate and a valid bearer token into the pod when it is instantiated.
+The `kubernetes` service (in all namespaces) is configured with a virtual IP address that is redirected (via kube-proxy) to the HTTPS endpoint on the apiserver.
+
+The control plane components also communicate with the cluster apiserver over the secure port.
+-->
+
+想要连接到 apiserver 的 Pods 可以使用服务账号安全地进行连接。
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:61]:
```suggestion
想要连接到 apiserver 的 Pod 可以使用服务账号安全地进行连接。
```

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+当 Pods 被实例化时，Kubernetes 自动把公共根证书和一个有效的持有者令牌注入到 Pod 里。
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:62]:
```suggestion
当 Pod 被实例化时，Kubernetes 自动把公共根证书和一个有效的持有者令牌注入到 Pod 里。
```

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+`kubernetes` 服务（位于所有名字空间中）配置了一个虚拟 IP 地址，用于（通过 kube-proxy）转发
+请求到 apiserver 的 HTTPS 末端。
+
+控制面组件也通过安全端口与集群的 apiserver 通信。
+
+<!--
+As a result, the default operating mode for connections from the nodes and pods running on the nodes to the control plane is secured by default and can run over untrusted and/or public networks.
+-->
+
+这样，从集群节点和节点上运行的 Pods 到控制面的连接的缺省操作模式即是安全的，能够在不可信的网络或公网上运行。
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:72]:
```suggestion
这样，从集群节点和节点上运行的 Pod 到控制面的连接的缺省操作模式即是安全的，能够在不可信的网络或公网上运行。
```

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+* Providing the kubelet's port-forwarding functionality.
+
+These connections terminate at the kubelet's HTTPS endpoint. By default, the apiserver does not verify the kubelet's serving certificate, which makes the connection subject to man-in-the-middle attacks, and **unsafe** to run over untrusted and/or public networks.
+-->
+
+### apiserver 到 kubelet
+
+从 apiserver 到 kubelet 的连接用于：
+
+* 获取 Pods 日志
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:102]:
```suggestion
* 获取 Pod 日志。
```

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+* 挂接（通过 kubectl）到运行中的 Pods
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:103]:
```suggestion
* 挂接（通过 kubectl）到运行中的 Pod。
```

File: content/zh/docs/concepts/architecture/control-plane-node-communication.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+
+最后，应该启用 [Kubelet 用户认证和/或鉴权](/docs/admin/kubelet-authentication-authorization/)来保护 kubelet API。
+
+<!--
+### apiserver to nodes, pods, and services
+
+The connections from the apiserver to a node, pod, or service default to plain HTTP connections and are therefore neither authenticated nor encrypted. They can be run over a secure HTTPS connection by prefixing `https:` to the node, pod, or service name in the API URL, but they will not validate the certificate provided by the HTTPS endpoint nor provide client credentials so while the connection will be encrypted, it will not provide any guarantees of integrity. These connections **are not currently safe** to run over untrusted and/or public networks.
+-->
+
+### apiserver 到节点、Pods 和服务
```
Comment (SataQiu) [inline content/zh/docs/concepts/architecture/control-plane-node-communication.md:133]:
```suggestion
### apiserver 到节点、Pod 和服务
```

**Comment by tengqm** [comment]
@SataQiu All feedbacks addressed. Force pushed revised content.

**Pull request merged**